### PR TITLE
Barcode resizing and text adjustment

### PR DESCRIPTION
**Pull request by Godmartinz**
Adjusts the size of the QR codes and repositions the barcode and associated text generated for labels

### CODE REVIEW

File: resources/views/hardware/labels.blade.php [hunk 1]
```diff
@@ -1,175 +1,168 @@
...
-    height: 100%;
-  }
-  img.barcode {
-    display: block;
-    margin-left: auto;
-    margin-right: auto;
-  }
-
- .qr_text {
-    width: {{ $qr_txt_size }}in;
```
Comment (snipe) [inline resources/views/hardware/labels.blade.php:61]: Does this mean we have a local variable(s) in this blade that we're not using anymore? If so, we should remove them.

**Review by snipe — CHANGES_REQUESTED** [review]
See inline comment

**Comment by snipe** [comment]
This is what I had before on `master`:

Screen Shot 2020-05-26 at 6 20 53 PM

This is what your branch is showing me:

Screen Shot 2020-05-26 at 6 31 13 PM

Is that the intended behavior, with the barcodes sitting right on top of each other?

**Comment by snipe** [comment]
Additionally, I'm not seeing the extra fields that were specified in the Admin Settings > Labels setting screen:

Screen Shot 2020-05-26 at 6 48 19 PM

(Sometimes this is simply because there isn't enough room on the barcodes though, so might not be a new artifact.)

File: resources/views/hardware/labels.blade.php (hunk 1 above)
Comment (Godmartinz) [reply resources/views/hardware/labels.blade.php:61]: removed

**Comment by Godmartinz** [comment]
label text has been considered! should be good to go now

**Comment by snipe** [comment]
Thanks - can you please make a PR for these changes to develop as well?

**Pull request merged**